Core/GDCore/Project/CustomObjectConfiguration.cpp

```diff
@@ -55,6 +78,13 @@ gd::ObjectConfiguration &CustomObjectConfiguration::GetChildObjectConfiguration(
   }
 
   auto &childObject = eventsBasedObject->GetObjects().GetObject(objectName);
+
+  if (!IsOverridingEventsBasedObjectChildrenConfiguration()) {
+    childObjectConfigurations.insert(
```

Comment:
This looks a tad weird. Do you think you could set up a log and see in which case this is used?

Reply:
When custom object follow their EBO, it's executed by:
- CustomObjectConfiguration::ExposeResources()
- RenderedCustomObjectInstance.getRendererOfInstance()
- RenderedCustomObjectInstance.getThumbnail()

`getRendererOfInstance` and `getThumbnail` are not run every frame so it should be ok.
`ExposeResources` is more risky, it may slow down refactor operations.

The constant version of the method is not actually used anywhere because `ExposeResources` need to be able to change resources names and the 2 other methods are JS side.